scipy/stats/morestats.py

```diff
@@ -2202,34 +2207,34 @@ def ansari(x, y, alternative='two-sided'):
     are different.
 
     >>> ansari(x1, x2)
-    AnsariResult(statistic=511.0, pvalue=0.3550608509212331)
+    AnsariResult(statistic=511.0, pvalue=0.3550608509212331)  # random
```

Comment:
I don't quite understand why we're adding the `random` in this PR even though the seed is still set above?

Reply:
> Will the seed be removed in another PR?

Yes. @tupui has been working on it in #13863.

> If so, why not add `random` in that PR?

Makes sense to do that. Is this OK for you @tupui? Otherwise, I am also fine with removing the seed here only.

Reply:
Actually the latest comments of the other PR points to still use a seed but to declare it in the outside in the refguide checker. This way the user is not faced with a value (which we do t want).

Since this will go in before. It does not matter here as I will rework it on the other side.

Reply:
> Since this will go in before. It does not matter here as I will rework it on the other side.

OK, then. I guess I will remove ` # random` for now. It can be tracked in #13863.

Reply:
We just merged #13863, then you might need to adapt the values. Run `python runtests.py -g -n --refguide-check` locally to get the correct values with the seeding.

Reply:
Is there a way to run just a particular file? `-t scipy.stats...` doesn't work like running a single file's tests.

Reply:
Just a single file I am not sure but you can do the module like this `python refguide_check.py --doctests stats`